tools/tpm2_rsaencrypt.c

```diff
@@ -46,14 +46,14 @@ typedef struct tpm_rsaencrypt_ctx tpm_rsaencrypt_ctx;
 struct tpm_rsaencrypt_ctx {
     struct {
         UINT8 k : 1;
-        UINT8 I : 1;
         UINT8 c : 1;
-        UINT8 unused : 5;
+        UINT8 unused : 6;
```

Comment:
Same comment than in previous commit, I think we can just drop this.

Reply:
Done

Reply:
Sorry this is from me working with Borland back in the day that would complain if your bitfield was not equal to the width of the type. It's just become habit to be explicit here, but it's not needed and ill stop now :-)